profile/README.md

```diff
@@ -0,0 +1,45 @@
+# Metal³
+
+The Metal³ project (pronounced: "Metal Kubed") provides components for bare
+metal host management with Kubernetes. You can enroll your bare metal machines,
+provision operating system images, and then, if you like, deploy Kubernetes
+clusters to them. From there, operating and upgrading your Kubernetes clusters
+can be handled by Metal³. Moreover, Metal³ is itself a Kubernetes application,
+so it runs on Kubernetes, and uses Kubernetes resources and APIs as its
+interface.
+
+Metal³ is one of the providers for the Kubernetes sub-project [Cluster
+API](https://github.com/kubernetes-sigs/cluster-api). Cluster API provides
+infrastructure agnostic Kubernetes lifecycle management, and Metal³ brings the
+bare metal implementation.
+
+Metal³ is open-source and welcomes community contributions.
+
+## Contacts
+
+* [Website](https://github.com/metal3-io)
```

Comment:
I would not say our website is organization page in Github, but https://metal3.io/ especially when this readme is being displayed in Github org page already :)

```suggestion
* [Website](https://metal3.io/)
```

Reply:
done thanks!